src/views/collection-view/server.ts

```diff
@@ -48,14 +63,6 @@ export async function getCollectionPageProps(
   slug: string
 ): Promise<{ props: CollectionPageProps }> {
   const collection = await getCollection(`${product.slug}/${slug}`)
-  // For sidebar data
-  const allProductCollections = await getAllCollections({
-    product: { slug: product.slug, sidebarSort: true },
-  })
-  const filteredCollections = filterCollections(
-    allProductCollections,
-    product.slug
-  )
   const layoutProps = {
```

Comment:
💭  We could add some of the sidebar layout props here? If we moved some of that level logic into `getStaticProps`

Reply:
I think we can construct some of the levels that'll get passed to the layout, but this view will have to be responsible for adding one more level before the array is passed, since it's setting a `children` prop that will be a React component. Same for `TutorialsView`, but that one will have to add 2 levels instead of just one. Side note, I think the same component can be used for both of those levels. I'll get that implemented in my next pass!